views/databaseIssue.ejs

```diff
@@ -0,0 +1,250 @@
+                <span>
+                    Docker mode:
+                    <span class="badge badge-dark"><%= isDocker ;%></span>
+                </span>
+                <span>
+                    Pm2 mode:
+                    <span class="badge badge-dark"><%= isPm2 ;%></span>
+                </span>
+                <span>
+                    Nodemon mode:
+                    <span class="badge badge-dark"><%= isNodemon ;%></span>
+                </span>
+                <span>
+                    OctoFarm version:
+                    <span class="badge badge-dark"><%= npmPackageJson ;%></span>
+                </span>
+                <span>
+                    NodeJS version:
+                    <span class="badge badge-dark"><%= nodeVersion ;%></span>
+                </span>
+                <span>
+                    OS:
+                    <span class="badge badge-dark"><%= os ;%></span>
+                </span>
+                <br />
+
+                <form method="post">
+                    <div class="form-group">
+                        <label for="connString">MongoDB url:</label>
+                        <div class="input-group">
+                            <div id="set-default-connection-button"
+                                 class="btn btn-secondary"
+                                 onclick="setDefaultMongoDbString()">
+                                Set default
+                            </div>
+                            <input type="url" id="connString" class="input-group-append form-control"
+                                   oninput="evaluateMongoDbInput()"
+                                   value="<%= mongoURL; %>"
+                                   placeholder="" />
+                        </div>
+                        <small id="connection-string-invalid" class="form-text text-danger">
+                            Your connection string seems to have invalid format. <br/>
+                            <ul>
+                                <li>Must start with <small class="badge">'mongodb://'</small></li>
+                                <li>Must contain <small class="badge">'/octofarm'</small></li>
+                                <li>
+                                    Read more about connection string in
+                                    <a href="https://docs.mongodb.com/manual/reference/connection-string/" target="_blank">
+                                        MongoDB Connection String URI Format documentation
+                                    </a>...
+                                </li>
+                            </ul>
+                        </small>
+                        <small id="connection-string-valid" class="form-text text-success">
+                            Your connection string seems properly formatted.
+                        </small>
+                        <!--                        <small id="usernamePasswordHelp" class="form-text text-muted">-->
```

Comment:
As we discussed removing redundant knowledge, this was commented out. Left here for the possibility of adding tooltips later.